File: SEBASTIAN_CHAN_00550218/Java/Mamifero/Gato.java
package Java.ProyectosSuperclass.Mamifero;


public class Gato extends Mamifero {
    private int nBigotes;
    
    
    
    public Gato(String nombre, double peso, int b){
        super(nombre,peso);
        this.nBigotes=b;
        
    }
    
    public void imprimirG(){
        System.out.println("Nombre del gato: " + this.getNombre());
        System.out.println("Peso: "+this.peso);
        System.out.println("Numero bigotes: " + this.nBigotes);
    }
    
}

File: SEBASTIAN_CHAN_00550218/C#/Animales/Gato.cs
using System;
public class Gato : Mamifero
{
    private int nBigotes;
    
    public Gato(string nombre, double peso, int b) : base(nombre, peso)
    {
        this.nBigotes = b;
    }
    
    public void ImprimirG()
    {
        Console.WriteLine("Nombre del gato: " + this.GetNombre());
        Console.WriteLine("Peso: "+ this.peso);
        Console.WriteLine("Numero bigotes: " + this.nBigotes);
    }
}
